impl/src/bitfield.rs

```diff
@@ -279,30 +278,29 @@ impl BitfieldStruct {
                 })
             }
 
-            let set_assert_msg = proc_macro2::Literal::string(
-                &format!("value out of bounds for field {}.{}", self.ast.ident, field_name)
-            );
+            let set_assert_msg = proc_macro2::Literal::string(&format!(
+                "value out of bounds for field {}.{}",
+                self.ast.ident, field_name
+            ));
```

Comment:
Could use `format_ident` here instead: https://docs.rs/quote/1.0.7/quote/macro.format_ident.html

Reply:
How'd you mean?

Reply:
yeah sorry doesn't make sense at this point. :sweat_smile: